modules/container-base/README.md

```diff
@@ -0,0 +1,56 @@
+# Dataverse Base Container Image
+
+A "base image" offers you a pre-installed and pre-tuned application server to deploy Dataverse software to.
+Adding basic functionality like executing scripts at container boot, monitoring, memory tweaks, etc., is all done
+at this layer, to make the application image focus on the app itself.
```

Comment:
Perhaps we should say that the application image is coming or planned. Make it clear that it's not yet available, I mean. Maybe we could link to an open issue.

Reply:
:heavy_check_mark: